src/test/java/DemoCode/Addition.java

```diff
@@ -0,0 +1,14 @@
+package DemoCode;
+
+public class Addition {
+
+	public static void main(String[] args) {
+		
+		int a = 10;
+		int b = 20;
+		
+		System.out.println(a+b);
```

Comment:
Do the addition of 3 number like a, b ,c
and print two statement like addition of 2 numbers and addition of 2 number

Reply:
added 3 numbers as aadition